libcxx/utils/libcxx/test/features.py

```diff
@@ -474,6 +476,39 @@ def _getAndroidDeviceApi(cfg):
     ),
 ]
 
+
+# Creation of symlinks require elevated privileges on Windows unless
+# Windows developer mode is enabled.
+def check_unprivileged_symlinks(cfg):
+    if not platform.system().lower().startswith("windows"):
+        return True
+
+    temp_file = tempfile.NamedTemporaryFile(delete=False)
```

Comment:
The comment I left below about host vs target distinction made me realize that this check is not implemented properly. You need to implement this check via e.g. `programSucceeds` (which will properly run the program on the target we're testing).